ts/src/pro/bitmart.ts

```diff
@@ -56,6 +57,9 @@ export default class bitmart extends bitmartRest {
                 'watchOrderBook': {
                     'depth': 'depth/increase100', // depth/increase100, depth5, depth20, depth50
                 },
+                'watchOrderBookForSymbols': {
```

Comment:
@ttodua can we infer the depth level from `limit`? if limit = 5, then uses depth5, etc,

Reply:
we need a bit massive rewrite to handle them better across all exchanges. so maybe we can retain this PR as is, and in future, i will do in batch for several exchanges at once, probably I'll not repeat the code and add helper in base

Reply:
what do you mean? I'm talking about this specific case and it should be relatively easy no? I did something similar on kucoin yesterday https://github.com/ccxt/ccxt/blob/master/ts/src/pro/kucoin.ts#L515

Reply:
ah, i misunderstood initially. ok

Reply:
@carlosmiei , after digging into that a bit more, i concluded that we should do that inference, because the different levels use the different endpoints, and those endpoints differ in functionality (one of that endpoint emits once in 100ms, while another emits once in every 500ms).
so, that option is more like "what speed of updates" user want to choose, and it shouldnt matter on `limit` value.
i have made last commit where I just documented it better, and I think this is final state of this PR atm

Reply:
sure no rpbolem